src/main/preprocessing/code_tracker_handler.py

```diff
@@ -45,16 +48,10 @@ def get_ct_language(data: pd.DataFrame) -> LANGUAGE:
 def handle_ct_file(ct_file: str) -> Tuple[pd.DataFrame, LANGUAGE]:
     log.info(f'Start handling the file {ct_file}')
     ct_df = pd.read_csv(ct_file, encoding=ISO_ENCODING)
-    # It's easier to have all tasks written in snake case because further they will be used as folders names and so on
-    ct_df[CODE_TRACKER_COLUMN.CHOSEN_TASK.value] = ct_df[CODE_TRACKER_COLUMN.CHOSEN_TASK.value].fillna(
-        ''). \
-        apply(lambda t: convert_camel_case_to_snake_case(t))
```

Comment:
why did you remove converting to camel case? I thought it was important to making plots and naming folders

Reply:
Sorry, I forgot that it is important. I have added it